### PR TITLE
[SRVKS-888]: Change the label used by Serverless

### PR DESCRIPTION
**Pull request by rh-max**
Version(s): 4.6+

Issue: https://issues.redhat.com/browse/SRVKS-888

Link to docs preview: https://deploy-preview-44824--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-ossm-setup.html#serverless-ossm-setup-with-kourier_serverless-ossm-setup

### CODE REVIEW

**Comment by rh-max** [comment]
@nak3 Hi Kenjiro, could you please review this? Thank you.

**Comment by nak3** [comment]
LGTM. Thank you @rh-max !

File: modules/serverless-ossm-setup-with-kourier.adoc [hunk 1]
```diff
@@ -72,13 +60,28 @@ spec:
...
 ----
 <1> Add the namespace that you want to integrate with {SMProductShortName}.
++
+[NOTE]
+====
+The `knative.openshift.io/part-of: "openshift-serverless"` label was added in {ServerlessProductName} 1.22.0. If you are using {ServerlessProductName} 1.21.0 or earlier, add the `knative.openshift.io/part-of` label to the `knative-serving` and `knative-serving-ingress` namespaces:
+
+[source,terminal]
+----
+$ oc label namespace knative-serving knative.openshift.io/part-of: "openshift-serverless"
```
Comment (jrangelramos) [inline modules/serverless-ossm-setup-with-kourier.adoc:77]: I think here it should be:
```suggestion
$ oc label namespace knative-serving knative.openshift.io/part-of=openshift-serverless
```

File: modules/serverless-ossm-setup-with-kourier.adoc [hunk 2]
```diff
@@ -72,13 +60,28 @@ spec:
...
+----
+
+[source,terminal]
+----
+$ oc label namespace knative-serving-ingress knative.openshift.io/part-of: "openshift-serverless"
```
Comment (jrangelramos) [inline modules/serverless-ossm-setup-with-kourier.adoc:82]: same to here:
```suggestion
$ oc label namespace knative-serving-ingress knative.openshift.io/part-of=openshift-serverless
```

**Review by jrangelramos — COMMENTED** [review]
just 2 minor nits.

File: modules/serverless-ossm-setup-with-kourier.adoc [hunk 3]
```diff
@@ -72,13 +60,28 @@ spec:
...
   podSelector: {}
   policyTypes:
   - Ingress
 ...
 ----
 <1> Add the namespace that you want to integrate with {SMProductShortName}.
++
+[NOTE]
+====
+The `knative.openshift.io/part-of: "openshift-serverless"` label was added in {ServerlessProductName} 1.22.0. If you are using {ServerlessProductName} 1.21.0 or earlier, add the `knative.openshift.io/part-of` label to the `knative-serving` and `knative-serving-ingress` namespaces:
```
Comment (pneedle-rh) [inline modules/serverless-ossm-setup-with-kourier.adoc:73]: Should "If you are using {ServerlessProductName} 1.21.0 or earlier" be "If you are using {ServerlessProductName} 1.21.0 or earlier"? Presumably the statement relates to all 1.21.x versions given that the label was not added until 1.22.0. I'm not familiar enough with Serverless to know if there were any z-stream versions after 1.21.0, but if there were, presumably they need to be included in this statement too.

File: modules/serverless-ossm-setup-with-kourier.adoc [hunk 4]
```diff
@@ -72,13 +60,28 @@ spec:
...
-          serving.knative.openshift.io/system-namespace: "true"
+          knative.openshift.io/part-of: "openshift-serverless"
   podSelector: {}
   policyTypes:
   - Ingress
 ...
 ----
 <1> Add the namespace that you want to integrate with {SMProductShortName}.
++
+[NOTE]
```
Comment (pneedle-rh) [inline modules/serverless-ossm-setup-with-kourier.adoc:71]: This note could be part of an additional step, given that it includes two procedural commands. For example, you could change it to a step with something like the following format:

> b.  If you are using OpenShift Serverless 1.21 or earlier, add the `knative.openshift.io/part-of` label to the `knative-serving` and `knative-serving-ingress` namespaces:
> 
> i. Add the label to the `knative-serving` namespace:
> 
> $ oc label namespace knative-serving knative.openshift.io/part-of=openshift-serverless
> 
> ii. Add the label to the `knative-serving-ingress` namespace:
> 
> $ $ oc label namespace knative-serving-ingress knative.openshift.io/part-of=openshift-serverless
> 
> **Note**: The knative.openshift.io/part-of: "openshift-serverless" label was added in OpenShift Serverless 1.22.0.

In the example step, I have also added substeps prior to each code block, to provide the reader with a clear understanding about what each command does explicitly.

**Review by pneedle-rh — COMMENTED** [review]
@rh-max I have now completed my review of this PR. Please let me know if you have any questions about my suggestions.

Please also squash the commits into one.

Thanks!

File: modules/serverless-ossm-setup-with-kourier.adoc (hunk 3 above)
Comment (rh-max) [reply modules/serverless-ossm-setup-with-kourier.adoc]: Well spotted, fixed it to say `1.21.1`.

**Comment by rh-max** [comment]
@jrangelramos Hi Jefferson, can you please give an explicit QE approve here? Thanks.

**Comment by jrangelramos** [comment]
QE Approved
LGTM

**Pull request merged**